gensim/models/base_any2vec.py

```diff
@@ -124,7 +124,17 @@ def _clear_post_train(self):
         raise NotImplementedError()
 
     def _do_train_job(self, data_iterable, job_parameters, thread_private_mem):
-        """Train a single batch. Return 2-tuple `(effective word count, total word count)`."""
+        """Train a single batch. Return 3-tuple
```

Comment:
Please follow [numpy-style](https://numpydoc.readthedocs.io/en/latest/) for docstrings, more links available [here](https://github.com/RaRe-Technologies/gensim/projects/4#column-1683544)